### PR TITLE
Input -> TextInput

### PR DESCRIPTION
**Pull request by ktj**
Default

Focus

Error

### CODE REVIEW

**Comment by ktj** [comment]
Should the text be red in error?

**Comment by ktj** [comment]
Merge #60 first

File: storybooks/native/themes/themes.hsl.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ const hslTheme = {
   quoteTextColor: colors.primary.hslGreyDark,
   letterSpacing: '-0.5px',
   activatablePointerHeight: '0.6rem',
-  activatableLineHeight: '4px'
+  activatableLineHeight: '4px',
+  error: '#dc0451',
```
Comment (mika-lindell) [inline storybooks/native/themes/themes.hsl.js]: I think these colors should come from ./color.js. I added misc-section into the feature/colors -branch

File: packages/hsl-shared-components/src/TextInput/Text.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ${props => props.theme.fontFamily && `font-family: ${props.theme.fontFamily};`}
+  color: ${props => props.theme.secondary};
+
+  padding: ${size(14)} ${size(18)};
+  border-radius: ${size(4)};
+  border: solid 1px ${props => lighten(0.185, props.theme.secondary)};
+  outline: 0;
+  transition: border-color 0.5s, color 0.5s;
+  &:focus {
+    color: #000000;
```
Comment (mika-lindell) [inline packages/hsl-shared-components/src/TextInput/Text.js]: The color used for focus in UIkit is not black, but dark shade of grey (#333).
-> `theme.colors.primary.hslGreyDark`

File: packages/hsl-shared-components/src/TextInput/Text.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { lighten } from 'polished';
+import { size } from '../utils';
+
+const Input = styled.input `
+  font-size: ${size(18)};
+  ${props => props.theme.fontFamily && `font-family: ${props.theme.fontFamily};`}
+  color: ${props => props.theme.secondary};
+
+  padding: ${size(14)} ${size(18)};
```
Comment (mika-lindell) [inline packages/hsl-shared-components/src/TextInput/Text.js]: I'm not sure, but to me it looks like there's just a bit too much padding here compared to UIkit. 
Maybe try something like `padding: ${size(14)} ${size(18)};`

File: storybooks/web/themes/themes.hsl.js [hunk 4]
```diff
@@ -27,7 +27,9 @@ const hslTheme = {
   quoteTextColor: colors.primary.hslGreyDark,
   letterSpacing: '-0.5px',
   activatablePointerHeight: '0.6rem',
-  activatableLineHeight: '4px'
+  activatableLineHeight: '4px',
+  error: '#dc0451',
```
Comment (mika-lindell) [inline storybooks/web/themes/themes.hsl.js]: I'd transfer these error colors to theme.colors -misc section

File: packages/hsl-shared-components/src/TextInput/Text.js (hunk 3 above)
Comment (ktj) [reply packages/hsl-shared-components/src/TextInput/Text.js]: When comparing images, then UI-kit seems to have less padding and larger font.
I'll reduce padding and increase font size slightly

**Pull request merged**